pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManagerTest.java

```diff
@@ -93,7 +92,8 @@
   private static final long TIMEOUT_IN_MS = 10_000L;
 
   @BeforeClass
-  public void setUp() throws Exception {
+  public void setUp()
```

Comment:
looks like the changes are mostly on formats. how about open a separate RP to land those first?

Reply:
Sure, will make a PR to only reformat